[2023-10-30T06:02:05.888+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-27T07:20:00+00:00 [queued]>
[2023-10-30T06:02:05.939+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-27T07:20:00+00:00 [queued]>
[2023-10-30T06:02:05.943+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 101
[2023-10-30T06:02:06.043+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_opensky_data> on 2023-10-27 07:20:00+00:00
[2023-10-30T06:02:06.066+0000] {standard_task_runner.py:57} INFO - Started process 109568 to run task
[2023-10-30T06:02:06.092+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'opensky_data_processing', 'fetch_opensky_data', 'scheduled__2023-10-27T07:20:00+00:00', '--job-id', '1030', '--raw', '--subdir', 'DAGS_FOLDER/DataFetcher.py', '--cfg-path', '/tmp/tmp3mxsrm_n']
[2023-10-30T06:02:06.106+0000] {standard_task_runner.py:85} INFO - Job 1030: Subtask fetch_opensky_data
[2023-10-30T06:02:06.385+0000] {task_command.py:416} INFO - Running <TaskInstance: opensky_data_processing.fetch_opensky_data scheduled__2023-10-27T07:20:00+00:00 [running]> on host ip-172-31-33-29.ec2.internal
[2023-10-30T06:02:06.907+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='adayani' AIRFLOW_CTX_DAG_ID='opensky_data_processing' AIRFLOW_CTX_TASK_ID='fetch_opensky_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-27T07:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-27T07:20:00+00:00'
[2023-10-30T06:03:33.011+0000] {local_task_job_runner.py:274} WARNING - Recorded pid 110642 does not match the current pid 109568
[2023-10-30T06:03:33.052+0000] {process_utils.py:131} INFO - Sending 15 to group 109568. PIDs of all processes in the group: [109568]
[2023-10-30T06:03:33.094+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 109568
[2023-10-30T06:03:33.094+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-10-30T06:03:33.095+0000] {logging_mixin.py:151} INFO - Error: Task received SIGTERM signal
[2023-10-30T06:03:33.097+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-30T06:03:33.244+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=opensky_data_processing, task_id=fetch_opensky_data, execution_date=20231027T072000, start_date=20231030T060330, end_date=20231030T060333
[2023-10-30T06:03:33.474+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=109568, status='terminated', exitcode=0, started='06:02:05') (109568) terminated with exit code 0
